app/src/main/java/com/fpvout/digiview/MainActivity.java

```diff
@@ -124,11 +162,84 @@ public void onClick(View v) {
         }
     }
 
+    private void toggleFullOverlay() {
+        if (overlayView.getAlpha() > 0.0f) return;
+
+        if (sharedPreferences.getBoolean(ShowWatermark, true)) {
+            toggleView(watermarkView, 0.3f);
+        }
+
+        settingsButton.removeCallbacks(hideSettingsButtonRunnable);
+        toggleView(settingsButton, new AnimatorListenerAdapter() {
+            @Override
+            public void onAnimationEnd(Animator animation) {
+                settingsButton.postDelayed(hideSettingsButtonRunnable, 3000);
+            }
+        });
+        liveButton.removeCallbacks(hideLiveButtonRunnable);
+        toggleView(liveButton, new AnimatorListenerAdapter() {
+            @Override
+            public void onAnimationEnd(Animator animation) {
+                liveButton.postDelayed(hideLiveButtonRunnable, 3000);
+            }
+        });
+    }
+
+    private void hideFullOverlay() {
```

Comment:
this is going to conflict with some of my work on refacto branch. We should discuss how to handle overlay in a better way, having all the logic in the main activity is kind of a pain...